prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -25,7 +25,8 @@ prow_ignored:
           cpu: "4000m"
     nodeSelector:
       # This job requires 8vCPUs or less, so it is "small".
-      cloud.google.com/gke-nodepool: small-job-pool
+      kpt-config-sync/type: periodic
```

Comment:
Are we labeling the CI nodes right now?

Reply:
We're using these labels for the nodes that the kind presubmits run on but not the other nodes yet. I'm adding labels to the other node pools

Reply:
ACK. We would need this to be documented in the playbook in case of recreating those nodepools.

Reply:
Sure, I can add a playbook